build.sh

```diff
@@ -97,6 +97,7 @@ fi
 if [ $do_clean = 1 -o $do_clean_bin = 1 ]; then              # Dont remove bin, as zsdcc and szdcpp must be built by hand in win32
   $MAKE clean
   $MAKE -C libsrc clean
+  $MAKE -C libsrc/_DEVELOPMENT clean
```

Comment:
libsrc/clean should clean libsrc/_DEVELOPMENT as well

Reply:
Confirmed. I see the top level Makefile does a clean in libsrc which does _DEVELOPMENT in the new-clean target. Backed out these 2 lines.